include/aikido/control/BarrettFingerKinematicSimulationPositionCommandExecutor.hpp

```diff
@@ -8,67 +8,74 @@
+class BarrettFingerKinematicSimulationPositionCommandExecutor
 {
 public:
   /// Constructor.
-  /// \param _finger Finger to be controlled by this Executor.
-  /// \param _proximal Index of proximal dof
-  /// \param _distal Index of distal dof
-  /// \param _collisionDetector CollisionDetector for detecting self collision
-  ///        and collision with objects. 
-  /// \param _collisionOptions Default is (enableContact=false, binaryCheck=true,
-  ///        maxNumContacts = 1.) 
-  ///        See dart/collison/Option.h for more information 
-  BarrettFingerPositionCommandExecutor(
-    ::dart::dynamics::ChainPtr _finger, int _proximal, int _distal,
-    ::dart::collision::CollisionDetectorPtr _collisionDetector,
-    ::dart::collision::CollisionOption _collisionOptions
+  /// \param[in] finger Finger to be controlled by this Executor.
+  /// \param[in] proximal Index of proximal dof
+  /// \param[in] distal Index of distal dof
+  /// \param[in] collisionDetector CollisionDetector to check collision with fingers.
+  ///        If nullptr, default to FCLCollisionDetector.
+  /// \param[in] collideWith CollisionGroup to check collision with fingers.
+  ///        If nullptr, default to empty CollisionGroup.
+  /// \param[in] collisionOptions Default is (enableContact=false, binaryCheck=true,
+  ///        maxNumContacts = 1.)
+  ///        See dart/collison/Option.h for more information
+  BarrettFingerKinematicSimulationPositionCommandExecutor(
+    ::dart::dynamics::ChainPtr finger, size_t proximal, size_t distal,
+    ::dart::collision::CollisionDetectorPtr collisionDetector = nullptr,
+    ::dart::collision::CollisionGroupPtr collideWith = nullptr,
+    ::dart::collision::CollisionOption collisionOptions
       = ::dart::collision::CollisionOption(false, 1));
 
   /// Sets variables to move the finger joints.
-  /// proximal dof moves to _goalPosition, joint limit, or until collision.
+  /// proximal dof moves to goalPosition, joint limit, or until collision.
   /// Step method should be called multiple times until future returns.
-  /// Distal dof follows with mimic ratio. 
-  /// \param _goalPosition Desired angle of proximal joint.
-  /// \param _collideWith CollisionGroup to check collision with fingers.
+  /// Distal dof follows with mimic ratio.
+  /// \param goalPosition Desired angle of proximal joint.
   /// \return future Becomes available when the execution completes.
-  std::future<void> execute(double _goalPosition,
-    ::dart::collision::CollisionGroupPtr _collideWith);
+  std::future<void> execute(double goalPosition);
 
-  /// Returns mimic ratio, i.e. how much the distal joint moves relative to 
+  /// Returns mimic ratio, i.e. how much the distal joint moves relative to
   /// the proximal joint. The joint movements follow
   /// this ratio only when both joints are moving.
   /// \return mimic ratio.
-  static double getMimicRatio();
+  constexpr static double getMimicRatio() { return kMimicRatio; }
```

Comment:
It seems that the only time that `getMimicRatio` is used is in tests, but `kMimicRatio` is private so getting rid of this function wouldn't work.